tests/unit/components/sl-menu-item-test.js

```diff
@@ -0,0 +1,141 @@
+import Ember from 'ember';
+import { moduleForComponent, test } from 'ember-qunit';
+
+moduleForComponent( 'sl-menu-item', 'Unit | Component | sl menu item', {
+    unit: true
+});
+
+test( 'Initial class names are present', function( assert ) {
+    this.subject();
+
+    assert.ok(
+        this.$().hasClass( 'sl-menu-item' ),
+        'Rendered element has class "sl-menu-item"'
+    );
+});
+
+test( 'Active state applies class', function( assert ) {
+    const component = this.subject();
+
+    assert.equal(
+        component.get( 'active' ),
+        false,
+        'Component is not active by default'
+    );
+
+    assert.equal(
+        this.$().hasClass( 'active' ),
+        false,
+        'Rendered element does not have class "active" by default'
+    );
+
+    Ember.run( () => {
+        component.set( 'active', true );
+    });
+
+    assert.ok(
+        this.$().hasClass( 'active' ),
+        'Rendered element has class "active" when component is active'
+    );
+});
+
+test( 'Class "has-sub-menu" is present when bound item has items array', function( assert ) {
+    const component = this.subject();
+
+    assert.ok(
+        false === this.$().hasClass( 'has-sub-menu' ),
+        'Rendered element does not have class "has-sub-menu" by default'
+    );
+
+    Ember.run( () => {
+        component.set( 'item', { items: [ { label: '' } ] } );
+    });
+
+    assert.ok(
+        this.$().hasClass( 'has-sub-menu' ),
+        'Rendered element has class "has-sub-menu"'
+    );
+});
+
+test( 'Class "active" is present when bound item is selected', function( assert ) {
+    const component = this.subject();
+
+    assert.ok(
+        false === this.$().hasClass( 'active' ),
+        'Rendered element does not have class "active" by default'
+    );
+
+    Ember.run( () => {
+        component.set( 'item', { selected: true } );
+    });
+
+    assert.ok(
+        this.$().hasClass( 'active' ),
+        'Rendered element has class "active"'
+    );
+});
+
+test( 'Component responds to mouse enter/leave events', function( assert ) {
+    const component = this.subject();
+
+    assert.ok(
+        false === component.get( 'active' ),
+        'Component is not active by default'
+    );
+
+    assert.ok(
```

Comment:
This is testing something already tested in the "Active state applies class" test.  Just testing the `active` property on the component will suffice.